backend/src/services/identity-project/identity-project-service.ts

```diff
@@ -226,7 +226,7 @@ export const identityProjectServiceFactory = ({
     if (!hasRequiredPriviledges)
       throw new ForbiddenRequestError({ message: "Failed to delete more privileged identity" });
 
-    const [deletedIdentity] = await identityProjectDAL.delete({ identityId });
+    const [deletedIdentity] = await identityProjectDAL.delete({ identityId }); // TODO: fix
```

Comment:
Regarding your "TODO: Fix" comment: What's wrong here, and should we fix it before merging?